tests/freertos/semaphore_test.h

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
The file has no header guards and no C binding section. Please refer the output of [TravisCI](https://travis-ci.org/RIOT-OS/RIOT/builds/509975119?utm_source=github_status&utm_medium=notification)